pkg/util/runtime/fuzz.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+// This file holds fuzzers that are implemented
+// using go-fuzz. More info about go-fuzz can
+// be found at https://github.com/dvyukov/go-fuzz
+
+// To run the below fuzzer locally, follow these steps:
```

Comment:
Sounds like something we would want integrated into our build Docker image, with an appropriate make target as well.

Reply:
It can be done. From my side, I will suggest doing that in a separate PR, as time does not permit me to get it done in this PR.